src/main/java/nl/tudelft/scrumbledore/game/GameFactory.java

```diff
@@ -24,26 +24,25 @@ public GameFactory() {
   }
 
   /**
-   * Make a new SinglePlayerGame.
+   * Makes a new Single / Multiplayer game.
+   * @param c 
+   *          A given class of game.
    * 
-   * @return The SinglePlayerGame.
+   * @return A SinglePlayer or MultiPlayer game.
    */
-  public Game makeSinglePlayerGame() {
-    Game game = new SinglePlayerGame();
+  public Game makeGame(Class c) {
```

Comment:
Checking a class or using intanceof should not be necessary in a good Factory Design Pattern right?